doc/subsystems/shell/shell.rst

```diff
@@ -168,6 +169,19 @@ that is found deepest in the command tree and further subcommands (without a
 handler) are passed as arguments. Characters within parentheses are treated
 as one argument. If shell wont find a handler it will display an error message.
 
+Commands can be also executed from user's application using DUMMY backend and
+a function :cpp:func:`shell_execute_cmd` like presented in below example:
+
+.. code-block:: c
+
+	void main(void)
+	{
+		/* Below code will execute "root_cmd subcommand" command */
+		shell_execute_cmd(NULL, "root_cmd subcommand param");
```

Comment:
This would be a better example if you showed a specific command and its parameters.